tests/testthat/test-BayesFactorBF.R

```diff
@@ -383,4 +383,28 @@ if (.runThisTest && requiet("testthat") && requiet("insight") && requiet("stats"
       tolerance = 1e-5
     )
   })
+
+
+  # ---------------------------
+  # Comes from https://github.com/easystats/bayestestR/issues/505
+
+  mtcars$cyl <- factor(mtcars$cyl)
+  mtcars$gear <- factor(mtcars$gear)
+  model <- lmBF(mpg ~ cyl + gear + cyl:gear, mtcars,
+                progress = FALSE, whichRandom = c("gear", "cyl:gear"))
+
+  test_that("find_formula for lmBF", {
+
+    predicted_form <- find_formula(model)$conditional
+    true_form <- formula(mpg ~ cyl + cyl:gear)
+
+    attr(predicted_form, ".Environment") <- NULL
+    attr(true_form, ".Environment") <- NULL
```

Comment:
I think you can use `ignore_attr = TRUE` in `expect_equal()`.

Reply:
Done